app/views/tasks/show.html.slim

```diff
@@ -160,9 +160,26 @@
             = link_to @task.license.link, @task.license.link
           - else
             = t('.no_license')
-
+    - if @task.contribution?
+      legend.toggle-next.closed
+        span.number = '3'
+        => t('tasks.shared.contribution_info')
+        span.fa-solid.fa-caret-down
+      .show-table.exercise-show style="display:none;"
+        .row
+          .col-auto.row-label
+            = TaskContribution.human_attribute_name('status')
+            | :
+          .col.row-value
+            = t("task_contributions.status.#{@task.task_contribution.status}")
+        .row
+          .col-auto.row-label
+            = Task.human_attribute_name('parent')
+            | :
+          .col.row-value
+            = link_to @task.parent.title, task_path(@task.parent)
     legend.toggle-next
-      span.number = '3'
+      span.number = '4'
```

Comment:
Does this mean a regular task has the numbers 1,2,4,5,... missing three?

Reply:
I have replaced the hard-coded value by an incremented variable. Therefore, this should no longer occur.